02-set-up.Rmd

```diff
@@ -17,7 +17,8 @@ Starting with the basics and moving to progressively more advanced topics, we ex
 R version, startup files and IDE can make your R work faster (though IDE could be seen as basic need for efficient programming).
 Ensuring correct configuration of these elements will have knock-on benefits in many aspects of your R workflow. That's why we cover them at this early stage (hardware, the other fundamental consideration, is covered in the next chapter).
 By the end of this chapter you should understand how to set-up your computer
-and R installation for optimal computational and programmer efficiency.
+and R installation (skip to section \@ref(r-version) if R is not already installed on your computer)
```

Comment:
Do we need to tell people to install R if they are reading this book? In theory they have already done Garett's course